src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -190,7 +189,7 @@ public HistoryPagePageObject openFileHistoryPage(String articlePage, String wiki
 
   public SignUpPageObject openSpecialSignUpPage(String wikiURL) {
     getUrl(wikiURL);
-    getGlobalNavigation().signUp();
+    //getGlobalNavigation().signUp();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)